parser-typechecker/src/Unison/Codebase/TranscriptParser.hs

```diff
@@ -175,11 +177,16 @@ run dir configFile stanzas codebase = do
                   -- invalid command is treated as a failure
                   Nothing ->
                     dieWithMsg
-                  Just pat -> case IP.parse pat args of
-                    Left msg -> do
-                      output $ P.toPlain 65 (P.indentN 2 msg <> P.newline <> P.newline)
-                      dieWithMsg
-                    Right input -> pure $ Right input
+                  Just pat -> 
+                    case IP.parse pat args of
+                      Left msg -> do
+                        output $ P.toPlain 65 (P.indentN 2 msg <> P.newline <> P.newline)
+                        let commands = IP.patternName <$> IPS.validInputs
+                        if IP.patternName pat `elem` ["help", "?", "help-topics"]
```

Comment:
Better to reference the fields in `IPS.help` and `IPS.helpTopics` than to hard-code strings here

Reply:
Good shout.